cmd/capabilities/dispatcher/dispatcher.go

```diff
@@ -0,0 +1,169 @@
+// Sample run-helloworld is a minimal Cloud Run service.
+package main
+
+import (
+        "log"
+        "time"
+        "os"
+        "context"
+        "runtime"
+        "encoding/json"
+        "cloud.google.com/go/pubsub"
+        "github.com/apigee/registry/rpc"
+	    "github.com/golang/protobuf/jsonpb"
+        "google.golang.org/grpc/codes"
+        "google.golang.org/grpc/status"
+        "github.com/apigee/registry/cmd/capabilities/utils"
+        cloudtasks "cloud.google.com/go/cloudtasks/apiv2"
+        taskspb "google.golang.org/genproto/googleapis/cloud/tasks/v2"
+)
+
+// Make this configurable
+const topicName = "changes"
```

Comment:
`changes` is the topic name is used by the registry backend to publish messages. I agree we can name this better, but it will be a bigger refactor. Will take care of this is a separate PR.

Reply:
Ack. Maybe better to add a TODO here or create a issue to keep track

Reply:
https://github.com/apigee/registry/issues/100